src/utils/css3-generator.js

```diff
@@ -47,16 +47,41 @@
 		return 'rotate' + prefix + theta + suffix;
 	};
 	
+	var getSkew = function (x, y, unit) {
+		if (!!!unit) {
```

Comment:
Maybe we should only have one `!`, just to make the code clearer to read 🤸‍♂

Reply:
fixer